pageserver/src/tenant/timeline.rs

```diff
@@ -3452,6 +3512,7 @@ impl Timeline {
         reconstruct_state: &mut ValuesReconstructState,
         cancel: &CancellationToken,
         ctx: &RequestContext,
+        read_path: &mut Option<ReadPath>,
```

Comment:
I would store this in `ValuesReconstructState` instead.

Reply:
fixed in https://github.com/neondatabase/neon/pull/10528/commits/9646db4a5744415e5de8c6dcc1da798e61118385